assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding; "This" is always refered to the window it is used for. So If this does not extend anything it will refer to the window. Or it will be undefined.
```

Comment:
"So If this does not extend anything it will refer to the window." It won't be undefined.